int16 filt_level
int16 arm_mode
int16 finger_mode
int16 neck_mode
float32[3] arx_pos_left # 0-2:pos_x, pos_y, pos_z
float32[3] arx_rot_left # 0-2:rot_x, rot_y, rot_z
float32[3] arx_pos_right # 0-2:pos_x, pos_y, pos_z
float32[3] arx_rot_right # 0-2:rot_x, rot_y, rot_z
float32[] hand_q_left
float32[] hand_q_right
float32[] neck_q